samples/newChat/css/style.css

```diff
@@ -0,0 +1,172 @@
+@import "./reset.css";
+@import "./login.css";
+@import "./dashboard.css";
+
+body, html {
+    height: 100%;
+    width: 100%;
+    background-color: #f0f2f4;
+    font-family: 'Roboto', sans-serif;
+    overflow: hidden
+}
+
+#page {
+    height: 100%;
+    width: 100%;;
+}
+
+/*
+ * SCROLLBAR CUSTOMIZATION
+ */
+
+::-webkit-scrollbar {
+    width: 5px;
+}
+::-webkit-scrollbar-track {
+    background-color: #ccc;
+    border-left: 1px solid #ccc;
+}
+::-webkit-scrollbar-thumb {
+    background-color: #1FBCD2;
+}
+::-webkit-scrollbar-thumb:hover {
+    background-color: #4054B2;
+    cursor: pointer;
+}
+/*::-webkit-scrollbar*/
+
+
+/* Animation */
+
+#fountainG{
+    flex: 0 0 102px;
+    position: relative;
+    width: 102px;
+    height: 12px;
+}
+
+.fountainG{
+    position:absolute;
+    top:0;
+    background-color:rgba(0,0,0,0);
+    width:12px;
+    height:12px;
+    animation-name:bounce_fountainG;
+    -o-animation-name:bounce_fountainG;
+    -ms-animation-name:bounce_fountainG;
+    -webkit-animation-name:bounce_fountainG;
+    -moz-animation-name:bounce_fountainG;
+    animation-duration:1.5s;
+    -o-animation-duration:1.5s;
+    -ms-animation-duration:1.5s;
+    -webkit-animation-duration:1.5s;
+    -moz-animation-duration:1.5s;
+    animation-iteration-count:infinite;
+    -o-animation-iteration-count:infinite;
+    -ms-animation-iteration-count:infinite;
+    -webkit-animation-iteration-count:infinite;
+    -moz-animation-iteration-count:infinite;
+    animation-direction:normal;
+    -o-animation-direction:normal;
+    -ms-animation-direction:normal;
+    -webkit-animation-direction:normal;
+    -moz-animation-direction:normal;
+    transform:scale(.3);
+    -o-transform:scale(.3);
+    -ms-transform:scale(.3);
+    -webkit-transform:scale(.3);
+    -moz-transform:scale(.3);
+    border-radius:8px;
```

Comment:
you don't need prefixes for border-radius http://caniuse.com/#search=border-radius